src/themes/dojo/variables.css

```diff
@@ -40,6 +40,7 @@
 	/* Composed values */
 	--title-bar-bg: var(--dojo-blue);
 	--title-bar-color: var(--dojo-white);
+	--title-padding: 5px;
```

Comment:
Nothing exists yet beyond `--container-padding`. I renamed this to `--header-padding` to be more generic.